### ISSUE TITLE
Homebrew installer marked as deprecated

### ISSUE THREAD

**Issue by kcirtapfromspace** [Rigellute/spotify-tui#1106]
Looks like in homebrew spotify-tui is marked as deprecated. Is this true?

**Comment by sewnie** [comment]
yes. if you look at the code page you can see that there have been no commits since 2021. this project is considered abandonware.

consider https://github.com/aome510/spotify-player/releases

**Comment by monfresh** [comment]
While it's true that there haven't been any commits in a couple of years, the reason why Homebrew marked spotify-tui as deprecated is because it uses OpenSSL 1.1, which has been deprecated. You can see this by running `brew info spotify-tui`.

**Comment by ed9w2in6** [comment]
@apprehensions Although I appreciate your effort in moving users to an actively maintained alternative with a certain level of feature parity, I disagree with your classification as abandonware, especially when the program still works and that the author is reachable.

As @monfresh have said the deprecation marking should be due to OpenSSL in which it will only affect builds.  It can be fixed by running `cargo update` (ref: https://github.com/Rigellute/spotify-tui/pull/1041).

**Comment by akankov** [comment]
Any plans to fix this?

**Comment by ed9w2in6** [comment]
@akankov Try if https://github.com/Rigellute/spotify-tui/pull/1041 fix the build process for you, if it does maybe give this a reaction to push the author to merge it.
